src/App.re

```diff
@@ -13,17 +13,30 @@ let reducer = (state, action) =>
   };
 
 [@react.component]
-let make = () => {
+let make = (~userName) => {
   let (state, dispatch) = React.useReducer(reducer, initialState);
+  
+  /*
+   * Create a style. Hyphenated CSS property names like margin-left become marginLeft.
```

Comment:
I was a little bit confused by the amount of comments in the current template, so I feel like it is better without a lot of comments, people could probably check the docs if they have trouble figuring smth out 😄 So I would remove comments.
What do others think?